dynatrace/api/v2/hub/extension/config/service.go

```diff
@@ -196,6 +196,9 @@ func (me *service) Create(ctx context.Context, v *extension_config.Settings) (*a
 }
 
 func (me *service) ensureInstalled(ctx context.Context, name string, version string) error {
+	if strings.HasPrefix(name, "custom:") {
```

Comment:
Would it make sense to have a check if it's installed before? Right now, we're assuming that every custom extension exists, right?
With the endpoint `GET /extensions/{extensionName}/{extensionVersion}`, we could check if a specific version of an extension is installed and we would only need to call the POST endpoint if it isn't installed and not a custom extension